chart/templates/promlens-depoloyment.yaml

```diff
@@ -0,0 +1,35 @@
+{{ if .Values.promlens.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-promlens
+  labels:
+    app: {{ template "tobs.fullname" . }}
+    chart: {{ template "tobs.chart" . }}
+    release: {{ .Release.Name }}
+    component: promlens
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "tobs.fullname" . }}
+      component: promlens
+  template:
+    metadata:
+      labels:
+        app: {{ template "tobs.fullname" . }}
+        component: promlens
+    spec:
+      containers:
+        - image: {{ .Values.promlens.image }}
+          imagePullPolicy: IfNotPresent
+          name: promlens
+          command: ["/promlens", "--web.default-prometheus-url", "{{ tpl .Values.promlens.defaultPrometheusUrl . }}" ]
```

Comment:
Should we use `--log.level=<severity-var>`, `--log.format=logfmt` and `--web.external-url=8080` (assuming if in future this changes in case)?

Reply:
added "--web.listen-address", ":8080" but I think others better kept default